STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.ui.graph.control.DescriptionTest.testGenerateDescription
MODIFIED FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/ui/graph/control/DescriptionTest.java
CLEANER: de.strullerbaumann.visualee.ui.graph.control.TestGroup100Case0.test247
CLEANER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/ui/graph/control/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 17
NEW CLEANER SIZE: 2
ELAPSED TIME: 66.09899997711182
==========================
@@ -50,0 +50,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
